includes/cli/class-co-authors-plus.php

```diff
@@ -132,6 +132,18 @@ function( $value ) {
 					$user_data['meta_input'][ $meta_key ] = $post_meta[ 'cap-' . $meta_key ];
 				}
 			}
+
+			// Check if a user with this email address already exists (they might be a Subscriber).
+			$user = get_user_by( 'email', $user_data['user_email'] );
+			if ( $user !== false ) {
+				if ( self::$verbose ) {
+					WP_CLI::line( sprintf( 'User with email %s already exists, email address will be updated.', $user_data['user_email'] ) );
```

Comment:
Non-blocking question: Should we include the updated email address in the line as well?

Reply:
Good call, added in 465fefce578a8a0d154d00c0ec503fb14ac97a7f